etc/add-license.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/sh
 
-copyright=$'// Copyright (C) MongoDB, Inc. 2017-present.
+copyright=$'// Copyright (C) MongoDB, Inc. 2022-present.
 //
 // Licensed under the Apache License, Version 2.0 (the "License"); you may
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 '
 
 add_copyright() {
-    local line=$(head -n 1 $1)
-    if [ "$line" == "// Copyright (C) MongoDB, Inc. 2017-present." ]; then
+    # Check if first 24 bytes match first 24 bytes of copyright notice.
+    local line=$(head -c 24 $1)
+    if [ "$line" == "// Copyright (C) MongoDB" ]; then
```

Comment:
Optional: Since this will make the correct copyright changes for any file, consider running this as part of the `sa-lint` CI task and fail if it made any changes to files.

E.g.
```bash
# Find all .go files not in the vendor directory.
find . -type f -name "*.go" -not -path "./vendor" | xargs ./etc/add-license.sh
# Check for any changes made; exit with a non-zero exit code if there is a diff.
git diff --quiet
```

That might also be a useful `make` task.